README.md

```diff
@@ -14,46 +14,67 @@ ____
-| Download |[![Download](http://i.imgur.com/odToka3.png)](https://github.com/tidusjar/Ombi/releases)             |      [![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/eap/artifacts)       |  [![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/dev/artifacts)            |[![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/DotNetCore/artifacts)            |
+| Download |[![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/DotNetCore/artifacts)            |      [![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/eap/artifacts)       |  [![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/dev/artifacts)            |
 # Features
-Here some of the features Ombi has:
-* All your users to Request Movies, TV Shows (Whole series, whole seaons or even single episodes!) and Albums
+Here some of the features Ombi V3 has:
+* Out of the box working on all OS Just download the zip and start Ombi!
+* Let users request Movies and TV Shows (Whole series, whole seaons or even single episodes!)
 * Easily manage your requests
 * User management system (supports plex.tv accounts and local accounts)
-* Sending newsletters
-* Fault Queue for requests (Buffer requests if Sonar/Couchpotato/SickRage is offline)
 * Allow your users to report issues and manage them separately
 * A landing page that will give you the availability of your Plex server and also add custom notification text to inform your users of downtime.
-* Allow your users to get notifications!
+* Allow your users to get custom notifications!
 * Secure authentication so you don't have to worry about those script kiddies
 * We check to see if the request is already in Plex, if it's already in Plex then why you requesting it?!
-* We have allowed the ability for a user to add a custom note on a request
 * It automatically updates the status of requests when they are available on Plex
 * Slick, responsive and mobile friendly UI
-
+* Ombi wil automatically update here self :)
+* Very fast system.
 
 ### Integration 
 We integrate with the following applications:
-* Plex server 1.2 (and higher)
-* Emby (beta)
+* Plex server
+* Emby
 * Sonarr
-* SickRage
-* CouchPotato
 * Radarr (beta)
-* Watcher (beta)
-* Headphones
+
 
 ### Notifications
 Supported notifications:
 * SMTP Notifications (Email)
+* Discord
+* Slack
 * Pushbullet
 * Pushover
-* Slack
-* Discord
-* Weekly Recently Added email notification to all of your Plex Users!
+* Mattermost
+
+### The difference between Version 3 and 2
+
+Over the last 7 months, we focust on the main functions on Ombi.
+Make it better, faster and more stable.
```

Comment:
Over the last 7 months, we focused on the main functions on Ombi, a complete rewrite while making it better, faster and more stable.

Reply:
not sure if it's a rewrite, that part could be removed

Reply:
It is a rewrite

Reply:
Done